pkgs/by-name/pi/picom-pijulius/package.nix

```diff
@@ -29,4 +30,11 @@ picom.overrideAttrs (previousAttrs: {
     homepage = "https://github.com/pijulius/picom";
     maintainers = with lib.maintainers; [ YvesStraten ];
   };
+
+  passthru.updateScript = nix-update-script {
```

Comment:
`unstableGitUpdater` would work better here, I believe.

Reply:
I'll try to use that instead, I couldn't check whether it worked last time as I was on darwin